tests/pytorch/test_pytorch_model_export.py

```diff
@@ -98,6 +98,7 @@ def get_subclassed_model_definition():
     can be invoked within a module to define the class in the module's scope.
     """
 
+    # pylint: disable=W0223
```

Comment:
Eventhough the `forward` function is implemented in the class, pylint is throwing the error - `W0223: Method '_forward_unimplemented' is abstract in class 'Module' but is not overridden (abstract-method)`. 

https://stackoverflow.com/questions/22186843/pylint-w0223-method-is-abstract-in-class-but-is-not-overridden - Disabled it as suggested in the link.